### DISCUSSION TITLE
LXC update script to include python repos

### DISCUSSION THREAD

**Discussion by urza-lab** [tteck/Proxmox#2121] [Ideas]
As certain LXCs are running based on packages from the python repositories (e.g. [changedetection](https://github.com/tteck/Proxmox/blob/main/install/changedetection-install.sh)), I had to idea to add the following to the update LXC script:

`pip3 show` piped into `pip3 install *** -U`

**Comment by urza-lab** [comment]
For the time being, I use the below as .sh within the LXC:

```
#!/bin/bash

# Update and upgrade system packages
apt update
apt upgrade -y

# Install or upgrade Python packages
pip3 install --upgrade changedetection.io playwright
```

**Reply by tteck** [reply]
Or, you could type `update` in the LXC console.

**Reply by urza-lab** [reply]
That returns `-bash: update: command not found`

**Reply by tteck** [reply]
oops... must be a pre Oct. 5 install, sorry.
You'll have to use the full command.
